packages/pwa-buildpack/lib/Utilities/loadEnvironment.js

```diff
@@ -103,7 +102,7 @@ function throwReport({ errors }) {
  */
 class Configuration {
     constructor(env, envFilePresent) {
-        this.env = env;
+        this.env = Object.assign({}, env);
```

Comment:
Can you just default the env param ie `constructor(env = {},...)`? This could have a side effect of spreading some value into the result object.

Reply:
The environment in a Configuration should be immutable, so I clone the `process.env` object here in order to prevent mutation from other references.

Reply:
Makes sense!